plugins/search-react/api-report.md

```diff
@@ -118,6 +147,7 @@ export const SearchContextProvider: (
 // @public
 export type SearchContextProviderProps = PropsWithChildren<{
   initialState?: SearchContextState;
```

Comment:
Oh.  What's the behavior of `initialState` when `parentContext` is set to `true`?

Reply:
It has been ignored, great that you asked.
There are two options I see:
- If the parent context is passed, do not accept the initial state (with conditional typing);
- This initial state should be updated to the parent state (this does not seem to be an expected behavior).
Do not accept the initial state is what I would recommend, what do you think?

Reply:
Done 🙌🏻